.github/ISSUE_TEMPLATE/Service_team_request.yaml

```diff
@@ -36,15 +41,19 @@ body:
     id: mini-api
     attributes:
       label: Minimum API Version Required
-      description: What is the minimum API version of your service required to implement your feature?
+      description: |
+        What is the minimum API version of your service required to implement your feature?
+        If you cannot provide, please fill in N/A.
       placeholder: e.g., 2021-04-01
     validations:
       required: true
   - type: textarea
     id: swagger
     attributes:
-      label: Swagger PR link
-      description: Provide a link to the location of your feature(s) in the REST API specs repo. If your feature(s) has corresponding commit or pull request in the REST API specs repo, provide them. This should be on the master branch of the REST API specs repo.
+      label: Swagger PR link / SDK link
+      description: |
+        Provide a link to the location of your feature(s) in the REST API specs repo. If your feature(s) has corresponding commit or pull request in the REST API specs repo, provide them. This should be on the master branch of the REST API specs repo.
```

Comment:
Should be "main" branch instead of "master" branch as "master" was already deprecated.